specs/Terminate Spec.md

```diff
@@ -0,0 +1,60 @@
+# **Terminate Program Spec**
+
+![MS PowerToys](https://hothardware.com/ContentImages/NewsItem/48038/content/Microsoft_PowerToys.jpg "PowerToys")
+## Shortcut for users to quickly terminate a running program
+### Authors: Benjamin Leverette and Prudence Phillips
+### Spec Status: Draft
+## 1. Overview
+
+**1.1. Elevator Pitch / Narrative** 
+
+Mike is debugging his code in Visual Studio and the program freezes with the "App not responding" text displaying on the title bar, informing him that he cannot continue his work without closing the program and restarting it. Mike can completely shutdown the app quicker using a Keyboard shortcut if he has this PowerToy installed on his computer.
+
+**1.2. Customers**
+
+PowerToys is mainly targeted towards Windows Power Users though it is available to users who want to experience using windows in a more efficient and productive way.
+ 
+  
+**1.3. Problem Statement and Supporting Customer Insights**
+
+Windows users need a quicker, more efficient way to completely shut down an app whenever it is unresponsive. The team is still required to find a proper name for this PowerToy.
+
+**1.4. Existing Solutions or Expectations**
+
+Users can already acheive the function of this PowerToy by accessing clicking End Task through Task Manager on Windows. Users need to be running Windows 10 and have PowerToys installed to use it.
+
+
+**1.5. Goals/Non-Goals**
+
+- Develop this PowerToy and have sufficient time for testing and integration within our assigned 8 weeks for the project. 
+
+- Come up with a suitable name for this PowerToy, preferrably one accepted by the community.
+
+## 2. Definition of Success
+
+**2.1. Expected Impact: Customer, and Technology Outcomes, Experiments + Measures**
+
+There is currently a sizable number of people who are following the PowerToys project. We hope to meet these people's expectations and receive good feedback from the community when we are done with this PowerToy.
+
+## 3. Requirements
+
+**3.1.	Functional Requirements**
+
+- A keyboard shortcut to display a confirmation window 
```

Comment:
What's the exact user experience of this option?
When I press the keyboard shortcut do I see the prompt immediately over whatever window my mouse is over? Over whatever Window I have focus on? Where exactly does the prompt show up in my screen?

Can we have the option to make it so when I hit the keyboard shortcut my mouse pointer turns into the red finger and I click on stuff it ends the program, and when I hit the keyboard shortcut again my mouse pointer turns back to normal? Can I disable the 'yes/no' prompt? 

I think this section of the spec needs detail, and you can get that by listing out _exactly_ how a user would interact with this Powertoy. 

For example:

---
To use the powertoy, a user will:
* Hit the chosen keyboard shortcut, their cursor will turn red indicating they are in 'Terminate' mode
* Now in Terminate mode, whenever they click on a Window it terminates that app instantly
   * Please note: Users can configure in their option settings to enable a 'yes/no' prompt for terminating apps
* Once finished terminating their apps, the user presses the keyboard shortcut again to leave 'Terminate' mode 
---

I prefer the above example as it very clearly goes over the user experience, and you can have multiple user experiences listed in your spec as people will prefer different things.